cpp/oneapi/dal/io/detail/load_graph.hpp

```diff
@@ -53,126 +54,127 @@ edge_list<std::int32_t> load_edge_list(const std::string &name) {
 
 template <typename Graph>
 void convert_to_csr_impl(const edge_list<vertex_type<Graph>> &edges, Graph &g) {
-    auto layout = oneapi::dal::preview::detail::get_impl(g);
-    using vertex_t = typename Graph::vertex_type;
-    using vector_vertex_t = typename Graph::vertex_set;
-    using vector_edge_t = typename Graph::edge_set;
-    using allocator_t = typename Graph::allocator_type;
-    using vertex_size_t = typename Graph::vertex_size_type;
-
     if (edges.size() == 0) {
```

Comment:
Is `edge_list<>` your own type or just an alias to some standard type like `std::vector`? 
If first, it is inconsistent with our codestyle - the method shall be named `get_size()` or `get_count()` if you are talking about the number of elements, not the number of bytes.

Reply:
It's an std::vector inside